lib/libzpool/util.c

```diff
@@ -158,3 +160,58 @@ show_pool_stats(spa_t *spa)
 
 	nvlist_free(config);
 }
+
+/*
+ * Sets given global variable in libzpool to given unsigned 32-bit value.
+ * arg: "<variable>=<value>"
+ */
+int
+set_global_var(char *arg)
+{
+	void *zpoolhdl;
+	char *varname = arg, *varval;
+	u_longlong_t val;
+
+#ifndef _LITTLE_ENDIAN
+	/*
+	 * On big endian systems changing a 64-bit variable would set the high
+	 * 32 bits instead of the low 32 bits, which could cause unexpected
+	 * results.
+	 */
+	fprintf(stderr, "Setting global variables is only supported on "
+	    "little-endian systems\n", varname);
```

Comment:
Almost there.  The build fails on big endian systems because this fprintf doesn't contain an extra `varname` variable without a format specifier.  You'll need to remove it to resolve the build, this code must have been added upstream without being built.

```
	fprintf(stderr, "Setting global variables is only supported on "
	    "little-endian systems\n");
```